frontend/src/component/ui/Table.tsx

```diff
@@ -1,22 +1,24 @@
 import React, { Component, ReactChild } from 'react';
 
 import './Table.scss';
+import classNames from 'classnames';
 
 interface IProps {
   headers?: ReactChild[];
   rows?: ReactChild[][];
+  colNames?: string[];
 }
 
 export class Table extends Component<IProps> {
-  private createHeaders(headers: ReactChild[] | undefined) {
+  private createHeaders(headers: ReactChild[] | undefined, colNames: string[] | undefined) {
     if (!headers || headers.length === 0) {
       return null;
     }
     return (
       <tr key={`header`}>
         {headers.map((cell: ReactChild, cellIndex: number) => {
           return (
-            <th key={`cell-${cellIndex}`} className="table-cell">
+            <th key={`cell-${cellIndex}`} className={!colNames ? "table-cell" : classNames("table-cell", colNames[cellIndex])}>
```

Comment:
Let's simplify this one. We want to remove table-cell and keep column class Names only.